internal/services/ecs/data_source_huaweicloud_compute_flavors.go

```diff
@@ -0,0 +1,116 @@
+package ecs
```

Comment:
Try to import from the Huawei cloud but not implementing it here.

Reply:
Ok